### ISSUE TITLE
Train LF Generative baseline

### ISSUE THREAD

**Issue by davidnvq** [davidnvq/visdial#5]
# Some notes: 
- The code can be replaced by our annotated version. 
- Should do some experiments to prove the correctness of the replaced version.
- Learning rate is fixed at `1e-3`. 
- Batch size for training is `32` while `validation` is `4` ( reduce from 5 -> 4)

# Configs file
[experiment.sh](https://github.com/quanguet/visdial/files/3159968/experiment.txt)

[train_experiment.py](https://github.com/quanguet/visdial/files/3159969/train_experiment.txt)
# Comet -ml
 [Comet - ml](https://www.comet.ml/lightcv/visdial-baselines/6a2689b9b46e4361bd5589a3fff52473)

# At epoch 6
![Some metrics](https://user-images.githubusercontent.com/30741785/57422365-bc12b600-724a-11e9-889b-e3671d4d7d71.png)

# From the pretrained model provided by authors
```
# Evaluate LF Generative on Val set after 20 epochs

r@1: 0.3813469111919403
r@5: 0.5668604373931885
r@10: 0.6262596845626831
mean: 19.782024383544922
mrr: 0.4756218492984772
ndcg: 0.5664547085762024
Writing ranks to /home/ubuntu/datasets/myvisdial/checkpoints/lf_gen/baseline/output/val_ranks.json
```
